src/index.ts

```diff
@@ -73,6 +73,18 @@ async function setupSysdigIntegration(): Promise<void> {
     return;
   }
 
+  const initialInterval: number = 20 * 60 * 1000; // 20 mins in milliseconds
+  setInterval(async () => {
```

Comment:
This should be `setTimeout` otherwise `setInterval` will make this repeat

Reply:
good catch!!